includes/class-sensei.php

```diff
@@ -685,18 +685,21 @@ public function install() {
 	 * Checks for plugin update tasks and ensures the current version is set.
 	 *
 	 * @since 2.0.0
+	 *
+	 * @deprecated $$next-version$$
 	 */
 	public function update() {
+		_deprecated_function( __METHOD__, '$$next-version$$' );
```

Comment:
@m1r0 do you remember, is it okay to use `$$next-version$$` here?

Reply:
Yep, that works. 👍